src/services/message.js

```diff
@@ -265,13 +265,11 @@ class MessageService {
     // if adminDMonly is false
 
     if (isAdmin) {
-      if (this.adminDMonly && this.convoType !== 'personal') {
+      if (this.adminDMonly && convoType !== 'personal') {
         const reply = `Hey admin commands should be sent in direct message to the bot, not in a room`
-        this.wickrAPI.cmdSendRoomMessage(this.messageService.vGroupID, reply)
+        this.wickrAPI.cmdSendRoomMessage(vGroupID, reply)
         return
-      }
-
-      if (!this.adminDMonly) {
+      } else if (this.adminDMonly) {
```

Comment:
What is the point of an "else if" when the previous if code does a "return" ?

Reply:
yeah probably redundant